pkg/util/kubernetes/apiserver/types.go

```diff
@@ -19,20 +19,15 @@ const (
 	autoscalersController controllerName = "autoscalers"
 	servicesController    controllerName = "services"
 	endpointsController   controllerName = "endpoints"
-	secretsController     controllerName = "secrets"
 )
 
 // InformerName represents the kubernetes informer names
-// TODO: make it private
 type InformerName string
 
 const (
-	nodesInformer     InformerName = "nodes"
 	endpointsInformer InformerName = "endpoints"
-	wpaInformer       InformerName = "wpa"
-	hpaInformer       InformerName = "hpa"
 	servicesInformer  InformerName = "services"
-	secretsInformer   InformerName = "secrets"
-	// TODO: make it private
-	PodsInformer InformerName = "pods"
+	SecretsInformer   InformerName = "secrets"
+	WebhooksInformer  InformerName = "webhooks"
+	PodsInformer      InformerName = "pods"
```

Comment:
Is it mandatory to have it public? 
due to the previous comment. I feel like we should have them private.

Reply:
They're imported by other packages, that's why they're public. I don't see how it can be harmful to expose them to other packages TBH.
Regarding the TODO it was discussed with container app and @vboulineau we agreed on having controllers in separate packages. Let me know what do you think of that.